guide/sharding/additional-information.md

```diff
@@ -40,7 +40,9 @@ In discord.js v12, <DocsLink path="class/ShardClientUtil?scrollTo=ids">`client.s
 :::
 
 ```js
-client.shard.broadcastEval('if (this.shard.ids.includes(0)) process.exit();');
+client.shard.broadcastEval(client => {
```

Comment:
There's a tip on line 39 that says "discord.js v12", should be updated to "discord.js v13"